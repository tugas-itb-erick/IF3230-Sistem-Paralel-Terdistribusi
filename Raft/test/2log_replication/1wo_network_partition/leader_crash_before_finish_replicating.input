5
set_timeout 1 0
simulate 10
client_send_request 1 + 5
client_send_request 1 * 5
client_send_request 1 - 2
crash 1
set_timeout 2 0
simulate 100
get_leader_state 2
get_data 2
get_data 3
get_data 4
get_data 5
restart 1
simulate 100
get_leader_state 2
get_data 1
get_data 2
get_data 3
get_data 4
get_data 5
end_simulation
