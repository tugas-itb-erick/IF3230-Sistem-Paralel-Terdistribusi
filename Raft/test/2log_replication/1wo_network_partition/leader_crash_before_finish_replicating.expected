L2 next_index[1 X 1 1 1 ] match_index[0 X 0 0 0 ]
0
0
0
0
L2 next_index[1 X 1 1 1 ] match_index[0 X 0 0 0 ]
0
0
0
0
0
